lib/package-manager.coffee

```diff
@@ -384,7 +384,13 @@ class PackageManager
   emitPackageEvent: (eventName, pack, error) ->
     theme = pack.theme ? pack.metadata?.theme
     eventName = if theme then "theme-#{eventName}" else "package-#{eventName}"
-    @emit eventName, pack, error
+    @emitter.emit eventName, pack, error
+
+  on: (selectors, callback) ->
+    subscriptions = new CompositeDisposable
+    selectors.split(" ").forEach (selector) =>
+      subscriptions.add @emitter.on selector, callback
+    subscriptions
```

Comment:
Normally, for objects that use `event-kit` and emit events, we define event-specific methods for subscribing, like this:

```coffee
onDidInstallAlternativePackage: (callback) ->
  @emitter.on('package-installed-alternative', callback)
```

We would just need to change callers of `PackageManager` to use this method, rather than `.on()`. Would you mind tweaking the style in this way?